includes/polling/ports.inc.php

```diff
@@ -362,6 +363,55 @@
         $port_stats = snmpwalk_cache_oid($device, 'cpeExtPsePortEntry', $port_stats, 'CISCO-POWER-ETHERNET-EXT-MIB');
     }
 }
+            
+//NAC Polling
+if ($config['enable_ports_nac']) {
```

Comment:
I was the one who said put this in ports but I didn't realise how much data this would pick up. I'd definitely agree this should be it's own polling module with the ability to call OS files specifically. Ideally, it would be done using OOP rather than the old code format.

Reply:
Ok, I told sometime to understand the code to do what I did, and to replace that to a module, I have to study more things on LNMS code. Im not sure if it will be quick, but I'll hard work on that.
Maybe we can release it as is, while I replace this to a module. What do you think?

Reply:
Done!
<img width="1422" alt="screen shot 2018-11-06 at 23 48 51" src="https://user-images.githubusercontent.com/15846428/48105431-aaf6d700-e21e-11e8-8d74-9319eb194726.png">

Reply:
Done.
<img width="1422" alt="screen shot 2018-11-06 at 23 48 51" src="https://user-images.githubusercontent.com/15846428/48105482-eb565500-e21e-11e8-93c9-dee553d71908.png">